setup.py

```diff
@@ -44,7 +44,7 @@
 DOWNLOAD_URL = 'https://github.com/rupertford/melody'
 DESCRIPTION = 'lightweight python parameter search tool'
 LONG_DESCRIPTION = open('README.md').read()
-LICENSE='OSI Approved :: BSD 3-Clause License'
+LICENSE = 'OSI Approved :: BSD 3-Clause License'
```

Comment:
Saw that you brought this trivial issue forward from the notes from the previous PR